modules/index.js

```diff
@@ -3,1729 +3,142 @@
+export { default as keys } from './keys.js';
+export { default as allKeys } from './allKeys.js';
+export { default as values } from './values.js';
+export { default as pairs } from './pairs.js';
+export { default as invert } from './invert.js';
+export { default as functions, default as methods } from './functions.js';
+export { default as extend } from './extend.js';
+export { default as extendOwn, default as assign } from './extendOwn.js';
+export { default as defaults } from './defaults.js';
+export { default as create } from './create.js';
+export { default as clone } from './clone.js';
+export { default as tap } from './tap.js';
+export { default as has } from './has.js';
+export { default as mapObject } from './mapObject.js';
+export { default as pick } from './pick.js';
+
+// Utility Functions.
+export { default as identity } from './identity.js';
+export { default as constant } from './constant.js';
+export { default as noop } from './noop.js';
+export { default as property } from './property.js';
+export { default as propertyOf } from './propertyOf.js';
+export { default as matcher, default as matches } from './matcher.js';
+export { default as times } from './times.js';
+export { default as random } from './random.js';
+export { default as now } from './now.js';
+export { default as escape } from './escape.js';
+export { default as unescape } from './unescape.js';
+export { default as templateSettings } from './templateSettings.js';
+export { default as template } from './template.js';
+export { default as result } from './result.js';
+export { default as uniqueId } from './uniqueId.js';
+export { default as chain } from './chain.js';
+export { default as iteratee } from './iteratee.js';
+
+// Function (ahem) Functions.
+export { default as partial } from './partial.js';
+export { default as bind } from './bind.js';
+export { default as bindAll } from './bindAll.js';
+export { default as memoize } from './memoize.js';
+export { default as delay } from './delay.js';
+export { default as defer } from './defer.js';
+export { default as throttle } from './throttle.js';
+export { default as debounce } from './debounce.js';
+export { default as wrap } from './wrap.js';
+export { default as negate } from './negate.js';
+export { default as compose } from './compose.js';
+export { default as after } from './after.js';
+export { default as before } from './before.js';
+export { default as once } from './once.js';
+
+// Find et al.
+export { default as findKey } from './findKey.js';
+export { default as findIndex } from './findIndex.js';
+export { default as findLastIndex } from './findLastIndex.js';
+export { default as sortedIndex } from './sortedIndex.js';
+export { default as indexOf } from './indexOf.js';
+export { default as lastIndexOf } from './lastIndexOf.js';
+export { default as find, default as detect } from './find.js';
+export { default as findWhere } from './findWhere.js';
+
+// Collection Functions (plus `omit`).
```

Comment:
Perhaps: "Collection functions work on both arrays and objects" (and Maps and Sets if/when we choose to support them)

Reply:
I should have specified what I was unhappy about. I'm fine with the "Collection Functions" part but not with the "(plus `omit`)" part. I felt compelled to append that because `omit` is out of place here, but it feels ugly.

I do like your suggestions to clarify the section titles a bit, though. I'll follow up on them.